scripts/install-windows-prereqs.ps1

```diff
@@ -648,7 +571,7 @@ function Install-DCAP-Dependencies {
         Throw "Failed to install nuget EnclaveCommonAPI"
     }
 
-    if (($LaunchConfiguration -eq "SGX1FLC") -or (${OS_VERSION} -eq "WinServer2019"))
+    if ($LaunchConfiguration -eq "SGX1FLC")
```

Comment:
Since we ran this code path automatically when running on 2019, should this check be removed?

Reply:
You could think of this check as to see if the user is setting up on ACC hardware, as opposed to a non-ACC vm as line 575 will not run on a non-SGX VM.

We never encountered this edge case before due to us always using host verification 2016 boxes (in which is actually a testing hole on our part). So the above would need to run when running a SGX1FLC setup but not a SGX1FLC-NoIntelDrivers.

I can add a comment regarding this as it's tribal info? but I am of the opinion the above implementation is correct.

Reply:
Works for me